tests/unit/mock-api-data/text_androids.json

```diff
@@ -0,0 +1,1149 @@
+[
+  {
+    "text": "do androids dream of electric sheep",
+    "score": null,
+    "pos_types": null,
+    "fingerprint": {
+      "positions": [
```

Comment:
Correct, but CorticalClient.getTextBitmap() appends these additional fields manually; this way both getTextBitmap() and getBitmap(), querying different API URLs, return dictionaries with common fields.